assets/js/index.js

```diff
@@ -103,6 +103,20 @@ Array.from(searchForms).forEach(container => {
   }
 });
 
+const menuPositioner = () => {
+  const nav = document.querySelectorAll("#sidebar-default")[1]
+  const itemActive = document.querySelectorAll(".sidebar-item-active")
```

Comment:
Amazing, thanks for figuring out a way to do this! I tried a couple times using the DOKS `scroll-lock.js` script but couldn't get it to work.

Would it be worth disabling/removing that DOKS script now? It lives here:
https://github.com/chainguard-dev/edu/blob/3a549f91fe7afd5bcec4f95c11527f97ed035ea3/layouts/partials/footer/script-footer.html#L60-L61

Reply:
Yeah, I just removed it.

Looked at `scroll-lock.js` and I think the issue is duplicated selectors over time in the theme partials. Aside: I have a preference to not using `sessionStorage` and setting position based on currently active item on page load.

Reply:
Yeah, using storage seems like a gross hack to me, I like how you've done this.